src/html/js/common.js

```diff
@@ -177,7 +207,19 @@ Common.getUnitUrl = function() {
 Common.changeLocalUnitToUnitUrl = function (cellUrl) {
     var result = cellUrl;
     if (cellUrl.startsWith(Common.PERSONIUM_LOCALUNIT)) {
-        result = cellUrl.replace(Common.PERSONIUM_LOCALUNIT + "/", Common.getUnitUrl());
+        if (Common.path_based_cellurl_enabled) {

```

Comment:
Not sure why you have to change it? Are you forcing all URL to use the new format?